src/common/sagas/authenticate.ts

```diff
@@ -0,0 +1,22 @@
+import {authUser, disauthUser} from 'common/actions/user';
+import {types} from 'common/constants/user';
+import {push} from 'connected-react-router';
+import {put, takeLatest} from 'redux-saga/effects';
+
+function* checkUserSaga() {
+	try {
+		const access_token = localStorage.getItem('access_token');
```

Comment:
for better working experience use a redux-localstorage.

Reply:
this enhancer helps to watch on all choosen parts of your state.

https://github.com/elgerlambert/redux-localstorage - see more below